stable/mysql/templates/deployment.yaml

```diff
@@ -3,10 +3,15 @@ kind: Deployment
 metadata:
   name: {{ template "mysql.fullname" . }}
   labels:
-    app: {{ template "mysql.fullname" . }}
     chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+    {{- if not .Values.deploymentLabels.app }}
+    app: {{ template "mysql.fullname" . }}
+    {{- end }}
+    {{- range $key, $val := .Values.deploymentLabels }}
+    {{ $key }}: {{ $val }}
```

Comment:
I guess you can just use toYaml here similar to podLabels and annotations further down in the file

Reply:
@olemarkus Thanks for the review! I've changed it.